js_modules/dagit/packages/core/src/types/globalTypes.ts

```diff
@@ -134,6 +134,7 @@ export enum RunStatus {
 
 export enum StepEventStatus {
```

Comment:
Not saying we should make this change here, but I found the name of this enum a little bit confusing when trying to grok this PR. It's the status of the step, right? Not the status of a particular step event?

Reply:
Yep, it's the status of a step. I agree that we should change this, but in a future PR